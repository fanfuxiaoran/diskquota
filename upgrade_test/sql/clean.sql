DROP TABLE IF EXISTS badquota.t1;
DROP TABLE IF EXISTS badbody_schema.t2;
DROP ROLE IF EXISTS testbody;
DROP SCHEMA IF EXISTS badquota;
DROP ROLE IF EXISTS badbody;
DROP SCHEMA IF EXISTS badbody_scehma;
DROP SCHEMA IF EXISTS deleteschema;
DROP SCHEMA IF EXISTS srs1;
DROP SCHEMA IF EXISTS srr1;
DROP SCHEMA IF EXISTS srE;
DROP SCHEMA IF EXISTS s1;
DROP SCHEMA IF EXISTS s2;
DROP SCHEMA IF EXISTS s3;
DROP SCHEMA IF EXISTS s4;
DROP SCHEMA IF EXISTS s5;
DROP SCHEMA IF EXISTS s6;
DROP SCHEMA IF EXISTS s7;

DROP TABLE IF EXISTS b;
DROP TABLE IF EXISTS b2;
DROP ROLE IF EXISTS srerole;
DROP ROLE IF EXISTS srole;
DROP ROLE IF EXISTS strole;
DROP ROLE IF EXISTS u1;
DROP ROLE IF EXISTS u2;
DROP ROLE IF EXISTS u3temp;

DROP EXTENSION diskquota;
