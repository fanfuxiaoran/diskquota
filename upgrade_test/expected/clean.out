DROP TABLE IF EXISTS badquota.t1;
DROP TABLE IF EXISTS badbody_schema.t2;
DROP ROLE IF EXISTS testbody;
DROP SCHEMA IF EXISTS badquota;
DROP ROLE IF EXISTS badbody;
DROP SCHEMA IF EXISTS badbody_scehma;
NOTICE:  schema "badbody_scehma" does not exist, skipping
DROP SCHEMA IF EXISTS deleteschema;
NOTICE:  schema "deleteschema" does not exist, skipping
DROP SCHEMA IF EXISTS srs1;
NOTICE:  schema "srs1" does not exist, skipping
DROP SCHEMA IF EXISTS srr1;
NOTICE:  schema "srr1" does not exist, skipping
DROP SCHEMA IF EXISTS srE;
NOTICE:  schema "sre" does not exist, skipping
DROP SCHEMA IF EXISTS s1;
NOTICE:  schema "s1" does not exist, skipping
DROP SCHEMA IF EXISTS s2;
NOTICE:  schema "s2" does not exist, skipping
DROP SCHEMA IF EXISTS s3;
NOTICE:  schema "s3" does not exist, skipping
DROP SCHEMA IF EXISTS s4;
NOTICE:  schema "s4" does not exist, skipping
DROP SCHEMA IF EXISTS s5;
DROP SCHEMA IF EXISTS s6;
NOTICE:  schema "s6" does not exist, skipping
DROP SCHEMA IF EXISTS s7;
DROP TABLE IF EXISTS b;
NOTICE:  table "b" does not exist, skipping
DROP TABLE IF EXISTS b2;
NOTICE:  table "b2" does not exist, skipping
DROP ROLE IF EXISTS srerole;
NOTICE:  role "srerole" does not exist, skipping
DROP ROLE IF EXISTS srole;
NOTICE:  role "srole" does not exist, skipping
DROP ROLE IF EXISTS strole;
NOTICE:  role "strole" does not exist, skipping
DROP ROLE IF EXISTS u1;
NOTICE:  role "u1" does not exist, skipping
DROP ROLE IF EXISTS u2;
NOTICE:  role "u2" does not exist, skipping
DROP ROLE IF EXISTS u3temp;
NOTICE:  role "u3temp" does not exist, skipping
DROP EXTENSION diskquota;
